<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>livraison</class>
 <widget class="QDialog" name="livraison">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>1281</width>
     <height>681</height>
    </rect>
   </property>
   <property name="title">
    <string>Gestion des livraisons</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1171</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="ajouter">
     <attribute name="title">
      <string>ajouter livraison</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ID livraison</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ID">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse livraison</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Adresse">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix livraison</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Prix">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>90</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="afficher">
     <attribute name="title">
      <string>afficher livraison</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>150</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier livraison</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>110</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer livraison</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>350</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri croissant par prix </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>410</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Export PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>460</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher geolocalisation</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>441</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ID1_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>280</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse livraison</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>240</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix livraison</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Prix_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>240</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Adresse_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>280</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>200</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ID livraison</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>410</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri croissant par adresse </string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>440</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri decroissant par adresse </string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>380</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri decroissant par prix </string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>60</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_recherche">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>380</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>350</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>entrez une id ,un prix ou une adresse pour faire la recherche</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>360</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Export Excel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>40</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
